### DISCUSSION TITLE
logfiles: where and how?

### DISCUSSION THREAD

**Discussion by richardm1** [Rem0o/FanControl.Releases#2638] [Q&A]
I was running the portable edition and there were logs in the main folder.  I have replaced this with the installable edition but it seems there is no logging.

Are logs configurable?  Verbosity?  Path?

My original problem is AMD RX570 sensors are no longer detected -- this was working yesterday but I must have broken something when I re-installed drivers and reconfigured my monitors.  Might there be a separate log facility for LibreHardwareMonitor?  HWinfo64 "sees" my RX570 sensors perfectly still.

**Comment by Rem0o** [comment]
logs are in the same "installation" folder as FanControl.   As for your RX570, make sure the GPU section of LHM is ticked, and ADLX is off. 
![image](https://github.com/user-attachments/assets/54b51ae5-b929-4c96-a6d0-3f5aed527f35)

**Reply by richardm1** [reply]
Hello, problem is fixed.  I used the well-known DDU (Display Driver Uninstaller) to wipe the drivers/configuration from my system. Then I reinstalled the latest Radeon driver kit.

I think the problem began when I stopped using my AMD iGPU (RDNA2-based) and switched to the "legacy" driver kit for Vega/Polaris. I didn't wipe everything in-between (like I should have).

Regarding logs: I've started FanControl.exe with ProcMon64 running and found it writing log.txt to systemroot\system32:

![image](https://github.com/user-attachments/assets/1d5e0147-9de8-4dcd-90a2-497119713df8)

Also, discovered a new issue I think?  When selecting Start At User Logon it creates a FanControl task that's trying to launch using a local account -- I logon with a domain account.  When running the task the symptom is EventID 332 Category _"Launch condition not met, user not logged-on"_:

_"Task Scheduler did not launch task "\FanControl"  because user "localpcname\localuser" was not logged on when the launching conditions were met. User Action: Ensure user is logged on or change the task definition to allow launching when user is logged off."_

I modified the task to RunAs my **domain** account and now it works. Perhaps it's getting confused because I have a local user account and a domain user account with the same name?

Shall I open an issue for this?
